src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
The 'else if' is unnecessary after an 'if' with a 'return' statement. You can remove the 'else' keyword and un-nest the second 'if' statement, which will simplify the logic and improve the readability of the code.